src/bantz/data/graph_store.py

```
@@ -0,0 +1,235 @@
+        """Increase edge weight when the relationship is used again.
+
+        Returns the new weight (capped at 1.0).
+        """
+        edges = await self.get_edges_by_id(edge_id)
+        if not edges:
+            return 0.0
+        new_weight = min(edges[0].weight + boost, 1.0)
+        await self.update_edge_weight(edge_id, new_weight)
+        return new_weight
+
+    async def get_edges_by_id(self, edge_id: str) -> List[GraphEdge]:
```

Ruff (0.15.0)

[warning] Unused method argument: `edge_id`

(ARG002)